### DISCUSSION TITLE
Pass 'owner_links' through default_args doesn't work

### DISCUSSION THREAD

**Discussion by Rattnik** [apache/airflow#32583] [General]
Hi, if I pass arguments to DAG through default_args, owner_links doesn't work
`
default_dag_args = {
    "queue": "default",
    "owner": "xxx",
    "owner_links": {"xxx": "https://google.com"}
}`
`dag = DAG("test_dag", description="test dag",
          schedule_interval="*/5 * * * *",
          start_date=datetime(2023, 6, 18),
          default_args=default_dag_args
          )`
but if i pass it in DAG operator, all the OK
`dag = DAG("test_dag", description="test dag",
          schedule_interval="*/5 * * * *",
          start_date=datetime(2023, 6, 18),
         owner_links= {"xxx": "https://google.com"}
          )`

**Comment by potiuk** [comment]
Not all parameters can be passed via default_args.. This is likely one of them

**Comment by josh-fell** [comment]
`owner_links` is a DAG-level parameter. You can think of `default_args` as "[default args for the tasks in this DAG](https://airflow.apache.org/docs/apache-airflow/1.10.6/tutorial.html?highlight=email#default-arguments)". So since `owner_links` is not defined at the task level, it's not applied as you'd hoped.
